mnist_cnn/mnist_cnn.cpp

```diff
@@ -177,7 +177,7 @@ int main()
               ens::ProgressBar(),
               // Stop the training using Early Stop at min loss.
               ens::EarlyStopAtMinLoss(
-                  [&](const arma::mat& /* param */)
+                  [&](const arma::mat&  /* param */)
```

Comment:
```suggestion
                  [&](const arma::mat& /* param */)
```

This was correct before.